### PR TITLE
Internationalization: Fix loading JSON files

### LINKED ISSUE #2946: Internationalization: sr-Cyrl.json should be sr-Cyrl-CS.json
## Version

`master` as of now. Targeting R8.

## Describe the bug

The strings from localization team use `sr-Cyrl-CS.json` instead of `sr-Cyrl.json`. We should use the correct JSON file.

Related files:
- `he-IL.js` to be removed
- `sl-SI.json` is the correct name, `sl-SL.json` to be removed
- `sr-Cyrl-CS.json` is the correct name, `sr-Cyrl.json` to be removed
- `sr-Latn-CS.json` is the correct name, `sr-Latn.json` to be removed
- `zh-CN.json` is the file for `zh-Hans`, `zh-Hans.json` to be removed
- `zh-HK.json` is the file for `zh-Hant-HK`
- `zh-TW.json` is the file for `zh-Hant`, `zh-Hant.json` to be removed

https://github.com/microsoft/BotFramework-WebChat/tree/master/packages/component/src/Localization

## Expected behavior

- Strings displayed in `sr-Cyrl` language will use the `sr-Cyrl-CS.json`
- Remove `sr-Cyrl.json` from the repository

## Additional context

[Bug]

### PR DESCRIPTION
**Pull request by compulim**
> Fixes #2946.

## Changelog Entry

-  Fixes [#2946](https://github.com/microsoft/BotFramework-WebChat/issues/2946). Updated JSON filenames for localization strings, by [@compulim](https://github.com/compulim) in PR [#2949](https://github.com/microsoft/BotFramework-WebChat/pull/2949)

## Description

Localization team use different JSON file names than Unicode standard. And we missed cleaning up, and got a bad naming.

## Specific Changes

Added tests to make sure the language code match the content of JSON files.

Localization team use different JSON file names than Unicode standard:

- `sr-Cyrl-CS.json` for `sr-Cyrl`
- `sr-Latn-CS.json` for `sr-Latn`
- `zh-CN` for `zh-Hans`
- `zh-HK` for `zh-Hant-HK`
- `zh-TW` for `zh-Hant`

Also, we have naming bugs:

- `sl-SL` should be `sl-SI`

And we have a bogus file:

- `he-IL.js` should be removed, use `he-IL.json` instead

The following language should default to RTL:

- `ar-SA`

---

-  [x] Testing Added

### CODE REVIEW

**Comment by compulim** [comment]
@corinagum can you verify if `ar-SA` should be default to RTL?

File: docs/LOCALIZATION.md [hunk 1]
```diff
@@ -18,11 +18,13 @@ To add a new language, please update the following files:
       -  `COGNITIVE_SERVICES_SPEECH_TO_TEXT` is used to indicate that the language is supported by Cognitive Services Speech-to-Text Service.
       -  `COGNITIVE_SERVICES_TEXT_TO_SPEECH` is used to indicate that the language is supported by Cognitive Services Text-to-Speech Service. If the service supports neural voices, set it to `"neural"`; otherwise, `true`
       -  [https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support](List of supported languages)
--  Update `/packages/component/src/Utils/normalizeLanguage.js` and add a normalization logic for ISO language.
+-  Update [`/packages/component/src/Utils/normalizeLanguage.js`](https://github.com/microsoft/BotFramework-WebChat/blob/master/packages/component/src/Utils/normalizeLanguage.js) and add a normalization logic for ISO language.
```
Comment (corinagum) [inline docs/LOCALIZATION.md]: Fix markdown link

File: docs/LOCALIZATION.md [hunk 2]
```diff
@@ -18,11 +18,13 @@ To add a new language, please update the following files:
       -  `COGNITIVE_SERVICES_SPEECH_TO_TEXT` is used to indicate that the language is supported by Cognitive Services Speech-to-Text Service.
       -  `COGNITIVE_SERVICES_TEXT_TO_SPEECH` is used to indicate that the language is supported by Cognitive Services Text-to-Speech Service. If the service supports neural voices, set it to `"neural"`; otherwise, `true`
       -  [https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support](List of supported languages)
```
Comment (corinagum) [inline docs/LOCALIZATION.md]: Fix markdown link

Comment (corinagum) [reply docs/LOCALIZATION.md]: image

**Pull request merged**